sirepo/package_data/static/js/madx.js

```diff
@@ -5,6 +5,7 @@ var srdbg = SIREPO.srdbg;
 
 SIREPO.app.config(function() {
     SIREPO.PLOTTING_COLOR_MAP = 'afmhot';
+    SIREPO.SINGLE_FRAME_ANIMATION = ['twissAnimation'];
```

Comment:
should this be in the schema or on the server?

Reply:
The schema seems fine. It isn't dynamic. @moellep thoughts?

Reply:
SINGLE_FRAME_ANIMATION is a UI constant. I don't think the server needs it.

Reply:
https://github.com/radiasoft/sirepo/issues/2687